src/pycram/external_interfaces/robokudo.py

```diff
@@ -1,135 +1,259 @@
-import sys
-from typing_extensions import Callable
-
 import rospy
 import actionlib
-import rosnode
 
 from ..designator import ObjectDesignatorDescription
 from ..datastructures.pose import Pose
-from ..local_transformer import LocalTransformer
-from ..datastructures.world import World
+from ..worlds.bullet_world import BulletWorld
 from ..datastructures.enums import ObjectType
+from typing import Any
+from geometry_msgs.msg import PoseStamped, PointStamped
 
-try:
-    from robokudo_msgs.msg import ObjectDesignator as robokudo_ObjetDesignator
-    from robokudo_msgs.msg import QueryAction, QueryGoal, QueryResult
-except ModuleNotFoundError as e:
-    rospy.logwarn(f"Could not import RoboKudo messages, RoboKudo interface could not be initialized")
+is_init = False
+
+
+def init_robokudo_interface():
```

Comment:
Please make this a decorator that can be directly attached to the respective methods that call RoboKudo, you can look in the dev branch for reference.